models/cards.js

```diff
@@ -304,6 +304,38 @@ Cards.attachSchema(
       optional: true,
       defaultValue: '',
     },
+    vote: {
```

Comment:
For these new database fields, please also add migrations to Cards table/collection to bottom of `wekan/server/migrations.js`.